www/js/recommendations.js

```diff
@@ -27,13 +31,16 @@ angular.module('emission.main.recommendations',['emission.services', 'emission.p
       $scope.currentDisplay = document.getElementById("reviewModal" + id);
       $scope.currentDisplay.style.display = "block";
     };
-    $scope.getDistance = function(x, y) {
-      var xDif = ($scope.xCoord - x) * 55;
-      var yDif = ($scope.yCoord - y) * 69;
-      var x2 = Math.pow(xDif, 2);
-      var y2 = Math.pow(yDif, 2);
-      var dist = Math.sqrt(x2 + y2);
-      return Math.round(dist * 10) / 10 + " miles away";
-    }
+
+    //waiting on Vanessa integration 
+    $scope.getSuggestion = function() {
+      return new Promise(function(resolve, reject) {
+        var msgFiller = function(message) {
+          console.log("THIS IS THE SUGGESTION" + JSON.stringify(message));
+          document.getElementById("0").innerHTML = JSON.stringify(message);
+        }
+        window.cordova.plugins.BEMServerComm.pushGetJSON("/suggestion_sys", msgFiller, resolve, reject);
```

Comment:
The general design pattern is to define this in `CommHelper` so that we have a list of all the remote calls in one place.

Reply:
```
$ grep -rl pushGetJSON www/js
www/js/services.js
```